doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -185,13 +204,14 @@ voxels that are very light on the EPI image.
 
 
 
-Common data preparation steps: resampling, smoothing, filtering
-----------------------------------------------------------------
+Common data preprocessing steps: resampling, smoothing, filtering
```

Comment:
We wanted to avoid the word `preprocessing` that is specific to operations applied before what we do here.